launcher/src/main/java/org/apache/spark/launcher/Main.java

```diff
@@ -101,6 +110,80 @@ public static void main(String[] argsArray) throws Exception {
     }
   }
 
+  private static void printUsage() {
```

Comment:
This duplicates all of the usage info in SparkSubmitArguments. We definitely don't want that.